spec/jekyll-admin/server/collection_spec.rb

```diff
@@ -32,56 +32,74 @@ def app
   end
 
   context "an individual collection" do
-    let(:documents) { last_response_parsed["documents"] }
-    let(:first_document) { documents.first }
-
     it "returns an individual collection" do
       get "/collections/posts"
       expect(last_response).to be_ok
       expect(last_response_parsed["label"]).to eq("posts")
     end
 
-    it "includes documents" do
+    it "doesn't includes documents" do
```

Comment:
I wanted to get your approval before updating them, but sure I can do it right now.

Basically, we are not exposing documents on `/:collection_name` endpoint anymore. This endpoint has only the information about the collection itself as it had in the first release. This could be beneficial in the long term that we might want to change the collection's attributes such as `label` and get only the necessary fields while doing it.

Related to this PR, we now don't have only documents but also have directories that are listed in index endpoints. Therefore putting them all under a field like `documents` or `entries` along with the collection info makes no sense. Instead we put them under their own endpoints. For example;

GET `/posts`
```javascript
{
"label": "posts",
"files": [],
"directory": "/Users/mertkahyaoglu/dev/jekyll-admin/spec/fixtures/site/_posts",
"relative_directory": "_posts",
"permalink": "/:categories/:year/:month/:day/:title:output_ext",
"http_url": null,
"api_url": "http://localhost:4000/_api/collections/posts"
}
```

GET `/posts/entries`, GET `/posts/entries/test`, etc.

```javascript
[
{
"name": "test",
"modified_time": "2017-02-02 17:51:11 +0200",
"path": "_posts/test",
"type": "directory",
...
},
{
"path": "_posts/2016-03-01-test.md",
"id": "/2016/03/01/test",
"relative_path": "_posts/2016-03-01-test.md",
"url": "/2016/03/01/test.html",
"collection": "posts",
...
}
]
```